LambtonMADT/myProjectName
# Add some action to the app

# About the Change

This change adds a new greeting to the app.

## How to test
1. Run the app
2. Add a name to the text field
3. Press the button
4. Verify if it showed the message "Hello <name>!"
